Makefile

```diff
@@ -1,5 +1,10 @@
 install:
-	sudo ln -fs $(shell pwd)/create-container.sh /usr/sbin/devenv
+	cp create-container.sh /usr/sbin/devenv
```

Comment:
What about updates? The `ln` hack was also allowing to just `git pull` the repo to get the latest updates.
With this approach (that I totally understand for officially released packages) the user will need 2 extra steps after a `git pull` in the repo: uninstall and install again. Am I missing something?

Reply:
> What about updates? The ln hack was also allowing to just git pull the repo to get the latest updates.

Oh, I didn't take that into consideration. Updates now would take 1 extra command, not 2, as `cp` overwrites silently by default:
`git pull && sudo make install`

Anyway, it's a matter of preferences, I see reasoning for doing it any of 2 ways. Would any of them be more coherent with the strategies of other pieces of software?

Reply:
In any case, we need to add the `update` step to the documentation